pkg/workload/workload.go

```diff
@@ -449,6 +449,10 @@ func HasAllChecks(wl *kueue.Workload, mustHaveChecks sets.Set[string]) bool {
 		return true
 	}
 
+	if len(mustHaveChecks) > len(wl.Status.AdmissionChecks) {
+		return false
+	}
```

Comment:
Why do we use `len(mustHaveChecks)` here? We use `mustHaveChecks.Len()` in other places such as L448, L460.

Reply:
I forgot it's a set :). Done.